ctapipe/reco/stereo_combination.py

```diff
@@ -107,12 +107,12 @@ class StereoMeanCombiner(StereoCombiner):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        supported = {
+        self.supported = {
```

Comment:
You might want to make self.supported by a dispatch dictionary, like this:
```py3
self.supported_combiner = {
  ReconstructionProperty.ENERGY: self._combine_energy   
  ReconstructionProperty.PARICLE_TYPE: self._combine_particle_type
  ReconstructionProperty.GEOMETRY: self._combine_geometry
}
```

Then it simplifies life a later since you can do something similar to 
```py3
# combine all properties that can be combined
for itm in ReconstructionProperty:
    if itm in self.supported_combiner and itm in self.property:
        self.supported_combiner[itm](event)
```

with no need to even make the the `properties` list comprehension or do any bitwise checking.   In fact, since you make `properties` a list anyhow, you aren'y really using the fact that `self.property` is a `Flag` enum